### DISCUSSION TITLE
How do we set up a AWS/Azure Gateway from MinIO in the k8s operator?

### DISCUSSION THREAD

**Discussion by buildgreatthings** [minio/minio#12902] [Q&A] [answered]
My use case is that I'm running several pods as batch jobs that retrieve data from S3, run a transformation, and then return the data back to the bucket. I am currently adding in the boto3 and azure libraries in each container, but I was interested in using the minio gateway as an alternative. 

The documentation for the minio k8s operator/tenant doesn't mention the gateway product. Is this feature no longer supported?

https://docs.min.io/docs/minio-gateway-for-s3.html

**Comment by harshavardhana** [comment, marked as answer]
It is not supported.
